src/components/toolbox/charts/index.js

```diff
@@ -2,39 +2,85 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useTheme } from '../../../utils/theme';
 import { CHART_TYPES } from '../../../constants/chartConstants';
```

Comment:
With your refactor, I think that `CHART_TYPES` is not needed anymore. You can directly
```js
import { Line, Bar, Doughnut } from 'react-chartjs-2';
```